configurator/backend/authorization/redis.go

```diff
@@ -455,6 +476,36 @@ func (rp *RedisProvider) deleteToken(tokenEntity *TokenEntity) error {
 		return err
 	}
 
+	_, err = conn.Do("HDEL", ssoTokensKey, tokenEntity.UserID)
```

Comment:
looks like original file was formatted badly.
Please enable in Goland
Tools - Actions on Save - Reformat code for Go files
and "Run gofmt" On code reformat. Type "Run gofmt" in search